kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/pneumaticcraft/assembly.js

```diff
@@ -0,0 +1,44 @@
+onEvent('recipes', (event) => {
+    if (global.isExpertMode == false) {
+        return;
+    }
+
+    recipes = [
+        {
+            input: 'thermal:machine_frame',
+            input_count: 2,
```

Comment:
see previous recommendation.

Reply:
This (and below) are actually actually pulled from @MuteTiefling's base script; I'll also update that while I'm at it.